public/index.html

```diff
@@ -0,0 +1,111 @@
+<!-- DOCTYPE HTML -->
+<html>
+<head>
+  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+  <title>Federated Wiki</title>
+</head>
+<body>
+
+<style>
+.lineup {
+  display: flex;
+  font-family: "Helvetica Neue", Verdana, helvetica, Arial, Sans;
+}
+
+.page {
+  font-size: 15px;
+  width: 275px;
+  margin: 8px;
+  padding: 20px;
+  border: black 1px dotted;
+}
+
+a {
+  text-decoration: none;
+}
+
+iframe {
+  width: 100%;
+}
+
+.missing {
+  background: #eee;
+  padding: 0.8em
+}
+
+.caption {
+  color: gray;
+  text-align: center;
+  text-size: small;
+}
+
+.surround {
+  background: #eee;
+  padding: 0.8em;
+}
+
+.surround img {
+  width: 100%;
+  margin-bottom: 10px;
+}
+
+.item.pagefold {
+  border-top: 2px solid lightgray;
+  clear: both;
+  height: 10px;
+  margin-top: 24px;
+  position: relative;
+  text-align: center;
+}
+
+.item.pagefold span {
+  background: white;
+  color: gray;
+  display: inline-block;
+  padding: 0px .5em;
+  position: relative;
+  top: -.8em;
+}
+.ready {
+  position: fixed;
+  top: 0;
+  left: 0;
+  padding: 5px;
+  background-color: white;
+}
+
+/* Portrait and Landscape - iPhone 6 */
+@media only screen
+  and (min-device-width: 375px)
+  and (max-device-width: 667px)
+  and (-webkit-min-device-pixel-ratio: 2) {
+  .lineup {
+    display: block;
+    position: relative;
+  }
+
+  .page {
+    position: absolute;
+    margin: 0px;
+    width: 85%;
+    background: white;
+  }
+
+  body {
+    background: green;
+    margin: 0px;
+  }
+}
+
+</style>
+
+<div id="content"></div>
+<script src="https://fb.me/react-0.14.6.js"></script>
+<script src="https://fb.me/react-dom-0.14.6.js"></script>
+<script src="http://fb.me/JSXTransformer-0.12.1.js"></script>
+<script src="https://code.jquery.com/jquery-3.1.0.js"></script>
+<script src="https://cdn.rawgit.com/showdownjs/showdown/1.4.3/dist/showdown.min.js"></script>
+<script type="text/jsx" src="index.js">
```

Comment:
There is no need for
```
<script src="https://fb.me/react-0.14.6.js"></script>
<script src="https://fb.me/react-dom-0.14.6.js"></script>
<script src="http://fb.me/JSXTransformer-0.12.1.js"></script>

<script src="https://cdn.rawgit.com/showdownjs/showdown/1.4./dist/showdown.min.js"></script>
```
as this is all dealt with by the build process.

Just need to keep the jquery load, for now, though I expect that will be replaced by using FETCH in due course.